### PR TITLE
[SEA-2023] Cancel DevOpsDays Seattle 2023

### PR DESCRIPTION
**Pull request by harbinger55**
Unfortunately, we had to make the decision to cancel this year to due to low ticket sales. All sponsors, speakers and ticket holders were notified on 7/13/2023 via email.

### CODE REVIEW

**Comment by glasnt** [comment]
There are some small metadata changes you'll also need to add: https://github.com/devopsdays/devopsdays-web/blob/main/utilities/docs/cancel-event.md specifically `cancel:yes` and `sponsors_accepted : "no"`

**Comment by mattstratton** [comment]
Eyeballing this - the sub-pages (sponsor, speakers, etc) will still be active and have likely been indexed by google. There’s a process for updates to cancel an event at https://github.com/devopsdays/devopsdays-web/blob/main/utilities/docs/cancel-event.md

if those changes are in this PR and somehow I didn’t see them, please disregard

**Comment by mattstratton** [comment]
@harbinger55 this PR against your branch should handle the things we mentioned
https://github.com/harbinger55/devopsdays-web/pull/1

**Comment by harbinger55** [comment]
Sorry was rushing and didnt realize there were instructions. I have updated the PR following the steps

File: content/events/2023-seattle/welcome.md [hunk 1]
```diff
@@ -1,40 +1,47 @@
 +++
 Title = "devopsdays seattle 2023"
 Type = "welcome"
-aliases = ["/events/2023-seattle/"]
+aliases = ["/events/2023-seattle/",
+"/events/2020-seattle/location/",
```
Comment (mattstratton) [inline content/events/2023-seattle/welcome.md:5]:
```suggestion
"/events/2023-seattle/location/",
```

File: content/events/2023-seattle/welcome.md [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
+"/events/2020-seattle/propose/",
```
Comment (mattstratton) [inline content/events/2023-seattle/welcome.md:6]:
```suggestion
"/events/2023-seattle/propose/",
```

File: content/events/2023-seattle/welcome.md [hunk 3]
```diff
@@ -1,40 +1,47 @@
...
+"/events/2020-seattle/registration/",
```
Comment (mattstratton) [inline content/events/2023-seattle/welcome.md:7]:
```suggestion
"/events/2023-seattle/registration/",
```

File: content/events/2023-seattle/welcome.md [hunk 4]
```diff
@@ -1,40 +1,47 @@
...
+"/events/2020-seattle/sponsor/",
```
Comment (mattstratton) [inline content/events/2023-seattle/welcome.md:8]:
```suggestion
"/events/2023-seattle/sponsor/",
```

File: content/events/2023-seattle/welcome.md [hunk 5]
```diff
@@ -1,40 +1,47 @@
...
+"/events/2020-seattle/program/",
```
Comment (mattstratton) [inline content/events/2023-seattle/welcome.md:9]:
```suggestion
"/events/2023-seattle/program/",
```

File: content/events/2023-seattle/welcome.md [hunk 6]
```diff
@@ -1,40 +1,47 @@
...
+"/events/2020-seattle/speakers/",
```
Comment (mattstratton) [inline content/events/2023-seattle/welcome.md:10]:
```suggestion
"/events/2023-seattle/speakers/",
```

**Comment by mattstratton** [comment]
Mostly good - need the proper year for the redirects but otherwise this is mergeable

so sorry this had to be canceled :/

**Comment by harbinger55** [comment]
doh! fixed the year. 

We will probably talk about canceling at the organizers summit :(

**Pull request merged**